packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/dataStreamSearch/dataStreamSearch.tsx

```diff
@@ -20,12 +18,28 @@ export function DataStreamSearch({ onSubmit, client }: DataStreamSearchProps) {
     defaultValues: { workspace: null, searchQuery: '' },
   });
 
+  const [isError, setIsError] = useState<boolean>();
+
   return (
     <SpaceBetween size='xxs'>
       <Header variant='h3' description='Search for modeled data streams by name or by the associated asset.'>
         Search modeled data streams
       </Header>
 
+      {isError && (
+        <SpaceBetween size='s'>
+          <Alert statusIconAriaLabel='Info' type='info' dismissible dismissAriaLabel='cancel'>
+            {infoMessage}
+          </Alert>
+          <Alert statusIconAriaLabel='Error' type='error' dismissible dismissAriaLabel='cancel'>
+            {errorMessage}
+            <Link external href='https://docs.aws.amazon.com/iot-twinmaker/latest/guide/twinmaker-gs-workspace.html'>
```

Comment:
add a white space after end of the error message. right now, it looks like 
`message.Learn` it should be `message. Learn`